### ISSUE TITLE
Uncaught Error: Call to undefined method

### ISSUE THREAD

**Issue by DavitGe** [htmlburger/carbon-fields#1066]
Hi,
I don't know if it's a bug or I'm doing something wrong. I tried to create container to my products with using carbon field but it shows that ->show_on_post_type('product') is undefined method. here's code:

 `<?php

if (!defined('ABSPATH')) {
  exit;
}

use Carbon_Fields\Container;
use Carbon_Fields\Field;

Container::make( 'product_options',  'products'  )
    ->show_on_post_type('product')

    ->add_tab('Product_Options'), [
        Field::make( 'text', 'product_price', 'price:' )
    ]);

?>
`

**Comment by Lupul** [comment]
Code formating is hard to follow in your post, this is how it should look 

```php
<?php

if (!defined('ABSPATH')) {
exit;
}

use Carbon_Fields\Container;
use Carbon_Fields\Field;

Container::make( 'product_options', 'products' )
->show_on_post_type('product')

->add_tab('Product_Options'), [
    Field::make( 'text', 'product_price', 'price:' )
]);
?>
```

Secondly, 'product_options' is not a valid container type, to add a container on product page you can do something like this:

```php
Container::make( 'post_meta', __('Product Page Settings') )
->where( 'post_type', '=', 'product' )
->add_fields([
    Field::make( 'text', 'cb_my_product_text_field', __( 'My product text field' ) )
        ->set_attribute( 'placeholder', 'Placeholder' )
]);
```

Also you have a syntax error when calling `add_tab` check the correct format here: https://docs.carbonfields.net/learn/containers/tabs.html

**Comment by DavitGe** [comment]
there was mistake in post type argument thanks a lot <#
